styles/base/main.scss

```diff
@@ -0,0 +1,64 @@
+body,
+header,
+section,
+footer,
+main,
+h1,
+h2,
+h3,
+h4,
+hr {
+    @include base-zero;
+}
+
+html {
+    font-size: 62.5%;
```

Comment:
It would be helpful if you added a comment for the font-sizes to how much 1 rem corresponds to e.g. in this case 1 rem = 10px